### PR TITLE
Feature: `checkmark-cta` block for the Substance3D Plugin pages

### PR DESCRIPTION
**Pull request by So1ene**
Resolves https://jira.corp.adobe.com/browse/MWPW-130573

* A `checkmark-cta` block for the Substance3D team to use in their Plugin Pages (an example of a plugin page that will soon be migrated to this franklin website: https://substance3d.adobe.com/plugins/substance-in-3ds-max/)
* The checkbox should enable the CTA when checked, and disable it when unchecked.

**Test URLs:**
- Before: https://main--cc--adobecom.hlx.page/drafts/solene/checkmark-cta/checkmark-cta?martech=off
- After: https://checkmark-cta--cc--wbstry.hlx.page/drafts/solene/checkmark-cta/checkmark-cta?martech=off

--
EDIT:
A working test URL of my previous commit: https://checkbox-cta--cc--wbstry.hlx.page/drafts/solene/checkmark-cta/checkmark-cta?martech=off (for previewing of the block until [Update utils.js #43](https://github.com/adobecom/cc/pull/43) gets merged to fix the milo libs import)
TEST URL: https://checkbox-cta--cc--wbstry.hlx.page/products/substance3d/plugins/substance-in-3ds-max

### CODE REVIEW

File: creativecloud/scripts/scripts.js [hunk 1]
```diff
@@ -132,7 +132,8 @@ async function loadArticlePromo(getMetadata) {
  * ------------------------------------------------------------
  */
 
-const miloLibs = setLibs(LIBS);
+/* eslint-disable-next-line import/prefer-default-export */
+export const miloLibs = setLibs(LIBS);
```
Comment (Ruchika4) [inline creativecloud/scripts/scripts.js:137]: I have updated the utils.js to fix the issue here getLibs was returning undefined in this PR https://github.com/orgs/adobecom/discussions/672 As it already has export the I think we can use that to get the libs. Once the above PR merged we can use below code in checkmark-cta and will not need to update scripts.js
Screen Shot 2023-04-27 at 7 04 56 PM
.js

**Review by Ruchika4 — CHANGES_REQUESTED** [review]

**Comment by Ruchika4** [comment]
@So1ene , could you please add the jira ticket also to the PR description

**Comment by Ruchika4** [comment]
@So1ene Please add jira ticket to the pr description.

**Comment by Ruchika4** [comment]
> Sorry here you go! https://jira.corp.adobe.com/browse/MWPW-130573

Thanks for sharing the jira ticket. I am not able to see a xd design for this block. Could you please share that too.

**Comment by aishwaryamathuria** [comment]
@So1ene 
Could you please confirm the use case for this block? Are there any substance3d pages currently migrated that need this block and was this created as a part of the same requirement?
Could we get a review on this from someone on substance team or Milo team who are aware of the the detailed use case?

File: creativecloud/blocks/checkmark-cta/checkmark-cta.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { getLibs } from '../../scripts/utils.js';
+
+const miloLibs = getLibs();
+const { createTag } = await import(`${miloLibs}/utils/utils.js`);
+const { decorateBlockAnalytics, decorateLinkAnalytics } = await import(`${miloLibs}/martech/attributes.js`);
+
+const checkmarkCTA = (block) => {
```
Comment (Ruchika4) [inline creativecloud/blocks/checkmark-cta/checkmark-cta.js:7]: Can we please have this function name as init to follow the milo nomenclature for blocks?

**Review by Ruchika4 — CHANGES_REQUESTED** [review]

**Comment by Ruchika4** [comment]
Closing this PR as requested by @williambsm

**Pull request closed without merging**